Pirate/Program.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+
+namespace Pirate
+{
+    class Program
+    {
+
+        static string GetUserResponse(string question)
+        {
+            Console.WriteLine(question);
+            string userResponse = Console.ReadLine();
+            while (userResponse == "")
+            {
+                Console.WriteLine("Please enter yee a response.");
+                Console.WriteLine(question);
+                userResponse = Console.ReadLine();
+            }
+            return userResponse;
+        }
+
+
+        static void Main(string[] args)
+        {
+
+
+            Console.WriteLine("ARRRR Welcome YEE to my Pirate Adventure!");
+            Console.WriteLine("");
+            string name = GetUserResponse("AAR what is your PIRATE name?");
+            Console.Clear();
+            string lastname = GetUserResponse("And what is yee Pirate last name?");
+            Console.Clear();
+            string newpiratename = (name + " " + lastname);
+
+            Console.WriteLine($"Welcome {newpiratename}, To my pirate adventure!");
+
+            string answer;
+
+            do
+
+            {
+
+                answer = GetUserResponse("Would you like to go on an adventure? Enter yes or no");
+                Console.Clear();
+
+                if (answer == "no")
+                {
+
+                    Console.WriteLine("That's an unnaceptable answer me matey!");
+                }
+
+                else if (answer == "yes")
+
+                {
+                    Console.WriteLine("PEERFECT LET'S BEGIN!!");
+                }
+            }
+            while (answer != "yes");
+
+
+
+            Console.WriteLine("I kneeeew yeee had it in ya!");
+
+            string answertwo;
+
+            do
+            {
+                answertwo = GetUserResponse("Would you like to search for buried treasure? Yes or No?");
```

Comment:
This variable name is hard to read. "answerTwo" would be better.
Also, when variables start having numbers, it's an indicator that their names could probably be better.  Consider something like "buriedTreasureResponse" or something similarly descriptive of its purpose